src/commands/send-request.js

```diff
@@ -249,6 +249,44 @@ const sendRequest = async (args) => {
     }
   }
 
+  // If returnResponse option is set, return response data instead of logging
```

Comment:
In this case I want response to be logged as well. Just return response always. No need for introducing 'returnResponse'